CHANGES.md

```diff
@@ -1,3 +1,39 @@
+## edge-19.8.3
+
+This edge release introduces a new `linkerd stat ts` subcommand, enabling
+retrieval of TrafficSplit metrics. It also introduces a
+`Kubernetes cluster monitoring` Grafana dashboard.
+
+* CLI
+  * Added TrafficSplit metrics via `linkerd stat ts` subcommand
+  * Fixed `linkerd uninject` not removing `linkerd.io/inject: enabled`
+    annotations
+  * Fixed `linkerd stat -h` example commands (thanks @ethan-daocloud!)
+* Proxy
+  * Added `request_handle_us` histogram to measure proxy overhead
+  * Updated Tap server to require identity
+  * Formatted log records consistently
+  * Fixed tap response streams never ending
+  * Fixed Destination service resource leak by hanging up destination
+    resolutions when dropped
+  * Authority label reflects logical dst
+  * Replaced futures_watch with tokio::sync::watch
```

Comment:
Not sure if any of these next 4 bullet points are user-facing enough to mention.

Reply:
Roll 'em up